### ISSUE TITLE
Update Calendar API for building dates

### ISSUE THREAD

**Issue by mikebarlow** [mikebarlow/MyCal#8]
Update calendar API to look and work nicer

```
$Dates = $Calendar->dates('2017-01-01', '2017-01-31')
    ->withEvents()
    ->get(); // returns date collection

echo $Calendar->buildTable($Dates);
```

**Comment by mikebarlow** [comment]
Links to #1

**Comment by mikebarlow** [comment]
Fixed in e3c8714901836c5eb17b276ace3207e398f5c94c
